lib/generators/statesman/templates/create_migration.rb.erb

```diff
@@ -1,9 +1,9 @@
-class Create<%= migration_class_name %> < ActiveRecord::Migration<%= "[#{ActiveRecord::Migration.current_version}]" if Statesman::Utils.rails_5_or_higher? %>
+class <%= migration_class_name %> < ActiveRecord::Migration<%= "[#{ActiveRecord::Migration.current_version}]" if Statesman::Utils.rails_5_or_higher? %>
   def change
     create_table :<%= table_name %> do |t|
       t.string :from_state, null: false
       t.string :to_state, null: false
-      t.text :metadata<%= ", default: #{metadata_default_value}" unless mysql? %>
+      t.<%= metadata_column_type %> :metadata<%= ", default: #{metadata_default_value}" unless mysql? %>
```

Comment:
nitpick/bikeshed: should we refactor the default value out into a method in `generator_helpers.rb` as well as `metadata_column_type`?